dune

```diff
@@ -0,0 +1,22 @@
+(library
+ (name zarith)
+ (wrapped false)
+ (flags -w -6)
+ (library_flags -cclib -lgmp)
```

Comment:
I would suggest to use `dune-configurator` here instead, having `-lgmp` as a fallback.
See https://github.com/ocaml/opam-repository/issues/18131